### DISCUSSION TITLE
Make Snippet view show all snippets

### DISCUSSION THREAD

**Discussion by neilernst** [James-Yu/LaTeX-Workshop#3348] [Q&A]
My snippet view currently only shows Symbols and TikZ commands. Is there a way to get the other snippets from https://github.com/James-Yu/LaTeX-Workshop/wiki/Snippets included in the list? In particular I keep forgetting Bold, Emph, etc. and it would be handy to have those.

**Comment by jlelong** [comment]
Unfortunately, it is not possible.
